pkg/gateway/nginx_test.go

```diff
@@ -53,6 +70,10 @@ http {
 }
 `
 	if want != string(got) {
-		t.Fatalf("unexpected output: want=\n%s\ngot=%s", want, string(got))
+		t.Fatalf(
+			"unexpected output: want=%sgot=%s",
+			strings.Replace(want, " ", "÷", -1),
```

Comment:
Simply there to make space differences more obvious in the resulting diff without introducing a dependency.